CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## next
```

Comment:
When we do this, can we keep `## next` so that the next PR doesn't have to add it back in?